_test/test_transformation/test_spher_proj.m

```diff
@@ -40,11 +40,27 @@ function test_coord_transf_4D_plus_offset(~)
 
             assertElementsAlmostEqual(pix0,pix_rec);
         end
+        function test_coord_transf_3D_plus_offset_throw_no_lattice(~)
+            proj = spher_proj('alatt',2*pi);
+            proj.offset = [1,2,3,4];
+            pix0 = [...
+                100,  0, 0 ,1;...
+                0  , 10, 0 ,1;...
+                0  ,  0, 1 ,1];
```

Comment:
If done without the `...` will automatically construct the 4x3 which may be more readable.
```suggestion
            pix0 = [100,  0, 0 ,1
                    0  , 10, 0 ,1
                    0  ,  0, 1 ,1];
```

Reply:
not sure -- I know that it will construct the matrix. I have intentially added semicoluns and dots to emphasise what kind of matrix it construct exactly for purpose of better readability